core/resources/shaders/polyline.vs

```diff
@@ -35,6 +36,18 @@ varying vec2 v_texcoord;
     varying vec4 v_lighting;
 #endif
 
+vec4 modelPosition() {
+    return a_position;
+}
+
+vec4 worldPosition() {
+    vec4 worldPosition = a_position * u_model + vec4(u_map_position.xy, 0., 1.);
```

Comment:
Here's a gotcha: since polylines have vertex positions at the "zero width" position, the world position should probably reflect the position after the vertex is extruded according to its rendered width. At least that's what I'm doing in JS, and I remember us fixing this because it was causing all lines that used world position to be a big smear. Not sure how that should be handled here since it's a bit trickier to do within a function.